src/HonzaBotner.Discord.Services/Commands/EmoteCommands.cs

```diff
@@ -16,7 +18,6 @@ namespace HonzaBotner.Discord.Services.Commands
     [Description(
         "Commands to display stats about emote usage. You can also use additional switches `animated` and `nonanimated`.")]
```

Comment:
Add mention about `debug`?

Reply:
I don't think that is really necessary, since this will be mostly for dev purposes and monitoring of performance and end-user should never use it.

Reply:
True.